Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [package]
 name = "librespot"
 version = "0.5.0-dev"
+rust-version = "1.62.1"
```

Comment:
The MSRV is currently 1.61: https://github.com/librespot-org/librespot/blob/dev/COMPILING.md

Reply:
Also I think this should be present in the `Cargo.toml` of all sub-packages.

Reply:
updated and added to all other `Cargo.toml` files